### DISCUSSION TITLE
"Sorry, something went wrong" error message when I move an issue from an archived repository between Project's Board columns

### DISCUSSION THREAD

**Discussion by vwheatle** [community/community#76385] [Projects and Issues]
### Select Topic Area

Bug

### Feature Area

Projects

### Body

I made a Project board for my organization's current work project which consisted of two repositories. Later into the project, we merged these repositories together, archiving the one that we moved things out of. The board still exists at the organizational level, despite it only really serving one repository.

The board had some leftover issues from the now-archived repository, and I tried moving them over into a new "indeterminate" column. The issues did not move into the column, and a "Sorry, something went wrong" message appeared in the corner.

A workaround is to outright remove the issue from one column and manually re-add it to the other.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by tanyatiwari995** [comment]
Possible Causes:
Archived Repository Restrictions

When a repository is archived, its issues become read-only. This means you can't modify them, move them, or reassign them to another column within a GitHub Project.

Project Board Sync Issues

Since the repository was archived, GitHub might not fully sync its issues with the project board anymore, leading to unexpected behavior.

Backend Limitations

GitHub’s internal handling of project issues might not allow movement of issues from an archived repository, causing the error.

Workarounds:
Manually Remove and Re-add the Issue

As mentioned, deleting the issue from the column and manually adding it back in is a workaround.

Restore the Archived Repository Temporarily

If possible, unarchive the repository, move the issues, and then re-archive it.

Duplicate the Issue in the Active Repository

Create a new issue in the active repository and link it to the original one before removing the old issue.

Check GitHub API or CLI for Workarounds

If you have access to automation, you might be able to use the GitHub API or CLI to move the issue programmatically.
